hack/build-wheels.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+ROOT_FOLDER="$(dirname "${0}")/.."
+
+if [ "$#" -ne 1 ]; then
+  echo 'Invalid number of arguments'
+  echo "Usage: ./build-wheels.sh <outputPath>"
+  exit 1
+fi
+
+_buildWheel() {
+  local _srcPath=$1
+  local _outputPath=$2
+  local _currentDir=$PWD
+
+  # Python really expects the `setup.py` to be on the current folder, so we'll
+  # `cd` into it and the go back again.
+  cd $_srcPath
+  python setup.py \
+    sdist -d $_outputPath \
+    bdist_wheel -d $_outputPath
+  cd $_currentDir
+}
+
+_main() {
+  local _outputPath="$PWD/$1"
```

Comment:
:confused: In [this line of the Dockerfile](https://github.com/SeldonIO/MLServer/pull/328/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R14) it looks like the given path is intended to be absolute, not relative